core/utils.py

```diff
@@ -41,11 +41,16 @@
 from typing import (
     IO, Any, BinaryIO, Callable, Dict, Iterable, Iterator, List, Optional,
     TextIO, Tuple, TypeVar, Union, overload)
+
 from typing_extensions import Literal
 
 _YAML_PATH = os.path.join(os.getcwd(), '..', 'oppia_tools', 'pyyaml-6.0')
 sys.path.insert(0, _YAML_PATH)
 
+_CERTIFI_PATH = os.path.join(
+    os.getcwd(), '..', 'oppia_tools', 'certifi-2021.10.8')
+sys.path.insert(0, _CERTIFI_PATH)
+
```

Comment:
Why is this added?

Reply:
For the certifi import on line 55

Reply:
But why wasn't it needed before?

Reply:
This was present earlier but I removed it by mistake in one of my commits.

Reply:
This was in originally in _utils.py_ at the same location where I have added it. I don't exactly remember when it got removed. I noticed it when CI tests started failing.

Reply:
But the diff is not showing that it was in this file, right? So it seems like it was in other file.

Reply:
@vojtechjelinek This is the commit where it got removed. It was in the same file i.e _utils.py_.

https://github.com/oppia/oppia/pull/15441/commits/a7ac1cc3fd62c5fee6f855681c3c7d1025891416#diff-2439d77444f0be435d92d3e5df78ab2f065592d4f66c075a1bc92e1b8bb41954